ubi9-openjdk-11.yaml

```diff
@@ -49,6 +49,7 @@ modules:
   - name: jboss.container.maven
     version: "3.8.11"
   - name: jboss.container.java.s2i.bash
+  - name: jboss.container.java.jlink
```

Comment:
If you add this as a dependency to jboss.container.java.s2i.bash instead, we can avoid having to add it to each image descriptor.